index.bs

```diff
@@ -1283,6 +1291,8 @@ partial interface HTMLIFrameElement {
         ::  |disposition|
         :   [=PermissionsPolicyViolationReportBody/allowAttribute=]
         ::  |allowAttribute|
+        :   [=PermissionsPolicyViolationReportBody/allowAttribute=]
```

Comment:
This should say `[=PermissionsPolicyViolationReportBody/srcAttribute=]`, I believe.

As a separate question, does including it here mean that these keys should exist in the report even if the values are null? Should we try to avoid that?

Reply:
Fixed!
We should avoid setting the key when allow attribute or src attribute is not set. But for potential permissions policy violation, it only triggers when there is a conflict between alllow attribute and permissions policy, so I think it will always be present.

Reply:
That makes sense - this section does only apply to potential violations.